### PR TITLE
Login page redesign

### PR DESCRIPTION
**Pull request by chriswongtv**

### CODE REVIEW

File: app/client/stylesheets/_home.scss [hunk 1]
```diff
@@ -71,6 +71,7 @@
 
 	.logo-header-container {
 		height: 125px;
+        margin-top: 0;
```
Comment (madhavgharmalkar) [inline app/client/stylesheets/_home.scss]: indent here

File: app/client/stylesheets/_login.scss [hunk 2]
```diff
@@ -1,9 +1,18 @@
...
+  &::before {
+    content: '';
+    background-image: linear-gradient(to bottom right, #02ccba 0%, #3d99be 31%, #9848d5 100%);
+    opacity: 0.8;
+    width: 100%;
+    height: calc(100% + 50px);
+    position: absolute;
+  }
 
   > .container {
```
Comment (madhavgharmalkar) [inline app/client/stylesheets/_login.scss:17]: no need for a ">" 
the sass compiler will add it in purely to the fact that it's nested

**Review by madhavgharmalkar — CHANGES_REQUESTED** [review]
pls fix <3

File: app/client/stylesheets/_home.scss (hunk 1 above)
Comment (chriswongtv) [reply app/client/stylesheets/_home.scss]: fixed

**Review by madhavgharmalkar — APPROVED** [review]

**Pull request merged**